README.md

```diff
@@ -868,9 +868,9 @@ _Libraries for building and using SQL._
 - [gorethink](https://github.com/dancannon/gorethink) - Go language driver for RethinkDB.
 - [goriak](https://github.com/zegl/goriak) - Go language driver for Riak KV.
 - [Kivik](https://github.com/go-kivik/kivik) - Kivik provides a common Go and GopherJS client library for CouchDB, PouchDB, and similar databases.
-- [mgm](https://github.com/kamva/mgm) - MongoDB model-based ODM for Go (based on official MongoDB driver).
-- [mgo](https://github.com/globalsign/mgo) - (unmaintained) MongoDB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
-- [mongo-go-driver](https://github.com/mongodb/mongo-go-driver) - Official MongoDB driver for the Go language.
+- [mgm](https://github.com/kamva/mgm) - DB model-based ODM for Go (based on official DB driver).
+- [mgo](https://github.com/globalsign/mgo) - (unmaintained) DB driver for the Go language that implements a rich and well tested selection of features under a very simple API following standard Go idioms.
+- [-go-driver](https://github.com/mongodb/mongo-go-driver) - Official MongoDB driver for the Go language.
```

Comment:
why did you remove the word mongo

Reply:
Sorry, I may have misoperated, I restore it